doc/articles/getting-started-counter.md

```diff
@@ -0,0 +1,322 @@
+
+
+# [Command Line](#tab/cli)
+
+> [!NOTE] 
+> If you don't have the Uno Platform dotnet new templates installed, follow [these instructions](xref:Uno.GetStarted.dotnet-new).
+
+From the command line, run the following command:
+
+```
+dotnet new unoapp -preset blank -presentation mvvm -markup xaml -o Counter
+```
+
+This will create a new folder called `Counter` containing the new application.
+
+If you want to discover all the options available in the `unoapp` template, run the following command:
+
+```
+dotnet new unoapp -h
+```
+
+Also, for more information on all the template options, see [Using the Uno Platform Template](xref:Uno.GettingStarted.UsingWizard)
+
+
+---
+
+At this point, you should select a target platform and run the application. 
+
+Keep the application running whilst completing the rest of this tutorial. Hot Reload is used to automatically update the running application as you make changes to both XAML and C# code. For more information on Hot Reload, see [Hot Reload](xref:Uno.Features.HotReload).
+
+## MainWindow and MainPage
+
+The majority of an Uno Platform application is defined in a class library project, in this case named `Counter`. This project contains the XAML files that define the layout of the application and the C# files that implement the application logic. There are also platform-specific projects, in the `Platforms`, folder contain the platform-specific code required to run the application on each platform.
+
+The startup logic for the application is contained in the `app.cs` file in the `Counter` project. In the `OnLaunched` method, the `MainWindow` of the application is initialized with a `Frame`, used for navigation between pages, and the `MainPage` is set as the initial page.
+
+The layout for the `MainPage` is defined in the `MainPage.xaml` file. This file contains the XAML markup that defines the layout of the application.
+
+```xml
+<Page x:Class="Counter.MainPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:Counter"
+      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+  <StackPanel
+        HorizontalAlignment="Center"
+        VerticalAlignment="Center">
+    <TextBlock AutomationProperties.AutomationId="HelloTextBlock"
+          Text="Hello Uno Platform"
+          HorizontalAlignment="Center" />
+  </StackPanel>
+</Page>
+```
+
+This defines a page with a background set to the theme resource `ApplicationPageBackgroundThemeBrush`, meaning it will adapt to the theme (`Dark` or `Light`) of the application. 
+
+The page contains a `StackPanel`, which will layout controls in a vertical stack, and is aligned in the center of the page, both horizontally and vertically. The `StackPanel` contains a single `TextBlock` control, which displays the text `Hello Uno Platform` and is aligned in the horizontal center of the `StackPanel`.
+
```

Comment:
```suggestion
This defines a page with a background set to the theme resource `ApplicationPageBackgroundThemeBrush`, meaning it will adapt to the theme (`Dark` or `Light`) of the application. 

The page contains a `StackPanel`, which will lay out controls in a vertical stack and is aligned in the center of the page, both horizontally and vertically. The `StackPanel` contains a single `TextBlock` control, which displays the text `Hello Uno Platform` and is aligned in the horizontal center of the `StackPanel`.

```